spring-addons-starter-oidc/src/main/java/com/c4_soft/springaddons/security/oidc/starter/SpringAddonsOAuth2LogoutRequestUriBuilder.java

```diff
@@ -30,9 +30,6 @@ public class SpringAddonsOAuth2LogoutRequestUriBuilder implements LogoutRequestU
     @Override
     public Optional<String> getLogoutRequestUri(ClientRegistration clientRegistration, String idToken, Optional<URI> postLogoutUri) {
         final var logoutProps = clientProperties.getLogoutProperties(clientRegistration.getRegistrationId());
-        if (logoutProps.map(OAuth2LogoutProperties::isEnabled).orElse(false)) {
```

Comment:
I think that this test should be kept: as [documented on the property](https://github.com/ch4mpy/spring-addons/blob/a2010649edf33a6b6bd5b7974f9eb7b1b0df55e9/spring-addons-starter-oidc/src/main/java/com/c4_soft/springaddons/security/oidc/starter/properties/SpringAddonsOidcClientProperties.java#L212) the aim is to give the opportunity to disable RP-Initiated Logout (keep the user session on the authorization server) and skip directly to the post-logout URI.

Reply:
True, it should be negated.

As a side note, I wonder how could the [resource-server_with_ui tutorial](https://github.com/ch4mpy/spring-addons/tree/master/samples/tutorials/resource-server_with_ui) ever work...